Resources/public/js/views/subitem/ez-subitemlistmoreview.js

```diff
@@ -83,6 +97,85 @@ YUI.add('ez-subitemlistmoreview', function (Y) {
             });
         },
 
+        /**
+         * Resets the selected status from columns
+         *
+         * @protected
+         * @method _resetSortClass
+         */
+        _resetSortClass: function() {
+            var columns = this.get('container').all('.ez-subitem-column-sortable');
+
+            columns.each( function (column) {
+                column.removeClass(COLUMN_SORT_ASC_CLASS);
+                column.removeClass(COLUMN_SORT_DESC_CLASS);
+            });
+        },
+
+        /**
+         * Adds the sort class according to the `sortCondition` attribute
+         *
+         * @protected
+         * @method _addSortClass
+         */
+        _addSortClass: function () {
```

Comment:
should probably be named `_addSortOrderClass` (same thing for the reset method, `_resetSortOrderClass`)